_posts/2014-02-15-geting-started-index.md

```diff
@@ -74,7 +74,7 @@ You'll also need to add the Sugar application to the `application/0` function in
 
 ```elixir
 def application do
-  [ applications: [ :sugar ],
+  [ applications: [],
```

Comment:
:+1: Agreed. I should have mentioned backwards compatibility before. Remember kids, typing with kids hanging on you is distracted typing. Don't be a distracted typist. 🙃